guest-rust/Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
 name = "guest"
 version = "0.1.0"
-authors = ["Karthik Ganeshram <karthik.ganeshram@fermyon.com>"]
+authors = ["The Spin authors"]
 edition = "2021"
-rust-version = "1.71"
+rust-version = "1.90"
```

Comment:
Do the sample guest and the `sdk` crate really need to be 1.90?  Spin needed that for the P3 stuff but that shouldn't impact cron guests should it?